controllers/serviceexport/controller.go

```diff
@@ -0,0 +1,143 @@
+package serviceexport
+
+import (
+	"time"
+
+	apicorev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/informers"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	discoveryinformers "k8s.io/client-go/informers/discovery/v1"
+	clientset "k8s.io/client-go/kubernetes"
```

Comment:
Renaming the `kubernetes` package from the `client-go` module  to `clientset` is actually a quite common practice in the Kubernetes client code, possibly because the name `kubernetes` is a little bit overreaching:

https://github.com/kubernetes/client-go/search?q=clientset

Reply:
then it should be fine. Sometime we will give the exception for the third party library if the package name is really confusing.